rero_ils/modules/api.py

```diff
@@ -319,3 +326,124 @@ def organisation_pid(self):
         if self.get('organisation'):
             return self.replace_refs()['organisation']['pid']
         return None
+
+
+class ElasticsearchRecord(dict):
```

Comment:
Why not using `invenio_record.api.RecordBase` as `class` and only surcharge the changed functionality?

Reply:
RecordBase should be use with SQLAchemy model. In this case, the idea is to have a Record class not linked to the database. In additon, if we want use RecordBase, all properties or methods must be surcharged. If is better, I will use RecordBase instead of dict